timeseries/src/autogluon/timeseries/trainer/abstract_trainer.py

```diff
@@ -461,66 +503,80 @@ def _train_multi(
                 if time_limit is not None:
                     time_start_model = time.time()
                     time_left = time_limit - (time_start_model - time_start)
+                    logger.info(f"Training time left: {time_left:.2f} seconds")
                     if time_left <= 0:
-                        logger.log(
-                            30, "Stopping training due to lack of time remaining"
+                        logger.info(
+                            f"Stopping training due to lack of time remaining. Time left: {time_left:.2f} seconds"
                         )
                         break
 
                 model_names_trained += self._train_and_save(
                     train_data, model=model, val_data=val_data, time_limit=time_left
                 )
 
+        logger.info("\nTraining complete")
+        logger.info(f"Models trained: {model_names_trained}")
+
+        try:
+            best_model = self.get_model_best()
+            logger.info(f"Best model: {best_model}")
+            logger.info(
+                f"Best model score: {self.get_model_attribute(best_model, 'val_score')}"
+            )
+        except ValueError as e:
+            logger.error(str(e))
```

Comment:
Is logger.error really what we want to be doing here? Does this lead to the ideal user experience?